src/subdatatable/subdatatable.jl

```diff
@@ -93,27 +93,22 @@ function SubDataTable(parent::DataTable, row::Integer)
     return SubDataTable(parent, [Int(row)])
 end
 
-function SubDataTable{S <: Integer}(parent::DataTable, rows::AbstractVector{S})
+function SubDataTable(parent::DataTable, rows::AbstractVector{<:Union{Integer, Null}})
```

Comment:
If we have that function, we don't need to keep the `view` method below which calls `any(isnull, rowinds)`: it will check for the presence of nulls once, then `convert` will check it again here.